packages/SqueakMaps-TiledMaps.package/SMATiledMapMorph.class/instance/getNameTo.With..st

```diff
@@ -1,8 +1,8 @@
 accessing
-getNameTo: aLatLong With: aZoom
+getNameTo: aLongLat With: aZoom
```

Comment:
Müsste das nicht `getNameTo: aPoint with: aNumber` sein?

Reply:
Siehe anderer Kommentar

Reply:
Wenn man den `aZoom` in `aNumber` umbenennt müsste man die Methodensignatur ändern, zu `getNameTo: aPoint withZoom: aNumber`. Ansonsten wird nicht klar, was `aNumber`macht.